### ISSUE TITLE
Not able to get any data from Contentful

### ISSUE THREAD

**Issue by Aleksefo** [EmaSuriano/gatsby-starter-mate#75]
Starting from yesterday, "develop" command works fine except when it comes to fetching contentful data. It doesn't get any data whatsoever, both in existing and fresh projects. Both locally and in Netlify builds. Other starters utilizing Contentful work just fine.

Steps to reproduce:
gatsby new mate-portfolio https://github.com/EmaSuriano/gatsby-starter-mate
npm run setup (with new Contentful project and access keys) 
yarn develop

```$ npm run prepare && gatsby develop
npm WARN lifecycle The node binary used for scripts is /var/folders/c4/xrr45lg54ld0r9xxwgv85xj80000gn/T/yarn--1572516991375-0.8536194620177942/node but npm is using /usr/local/Cellar/node/12.12.0/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> gatsby-starter-mate@2.5.1 prepare /Users/aleksefo/Projects/mate-new
> npm run fix-medium-plugin && gatsby clean

npm WARN lifecycle The node binary used for scripts is /var/folders/c4/xrr45lg54ld0r9xxwgv85xj80000gn/T/yarn--1572516991375-0.8536194620177942/node but npm is using /usr/local/Cellar/node/12.12.0/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> gatsby-starter-mate@2.5.1 fix-medium-plugin /Users/aleksefo/Projects/mate-new
> sed -i' ' 's/\/latest//g' ./node_modules/gatsby-source-medium/gatsby-node.js

info Deleting .cache, public
info Successfully deleted directories
success open and validate gatsby-configs - 0.031s
success load plugins - 0.030s
success onPreInit - 0.003s
success initialize cache - 0.009s
success copy gatsby files - 0.048s
success onPreBootstrap - 0.008s
success source and transform nodes - 0.033s
success building schema - 0.151s
success createPages - 0.002s
success createPagesStatefully - 0.047s
success onPreExtractQueries - 0.002s
success update schema - 0.026s

 ERROR #85901  GRAPHQL
There was an error in your GraphQL query:
Unknown type "ContentfulFixed".
File: node_modules/gatsby-source-contentful/src/fragments.js:20:37

// Many similar errors

 ERROR #85901  GRAPHQL
There was an error in your GraphQL query:
Unknown type "ContentfulSizes".
File: node_modules/gatsby-source-contentful/src/fragments.js:20:55

failed extract queries from components - 0.590s
success write out requires - 0.027s
success write out redirect data - 0.002s
success onPostBootstrap - 0.001s
⠀
info bootstrap finished - 2.722 s
⠀
success run queries - 0.028s - 4/4 145.28/s
⠀
You can now view gatsby-starter-mate in the browser.
```

After that the GraphQL shows zero signs of Contentful:
![Screenshot 2019-10-31 at 12 20 26](https://user-images.githubusercontent.com/18084119/67938851-f9904480-fbd8-11e9-9378-e6c17c364ca0.png)

Removing the content of node_modules/gatsby-source-contentful/src/fragments.js to get to the actual issue and running "develop" again returns:

```info Deleting .cache, public
info Successfully deleted directories
success open and validate gatsby-configs - 0.029s
success load plugins - 0.029s
success onPreInit - 0.003s
success initialize cache - 0.007s
success copy gatsby files - 0.032s
success onPreBootstrap - 0.007s
success source and transform nodes - 0.026s
success building schema - 0.136s
success createPages - 0.001s
success createPagesStatefully - 0.044s
success onPreExtractQueries - 0.001s
success update schema - 0.019s

 ERROR #85907  GRAPHQL

There was an error in your GraphQL query:

- Unknown field 'contentfulAbout' on type 'Query'. Source: document `FooterQuery` file: `GraphQL request`

File: src/sections/Writing.js

failed extract queries from components - 0.265s
success write out requires - 0.030s
success write out redirect data - 0.002s
success onPostBootstrap - 0.002s
⠀
info bootstrap finished - 2.234 s
⠀
success run queries - 0.024s - 4/4 167.58/s
⠀
You can now view gatsby-starter-mate in the browser.
```

It started happening out of the sudden. At first, I assumed I messed up some local setup, but Neflify builds are failing too.

**Comment by Aleksefo** [comment]
The issue happens due to latest "gatsby": "2.17.7", setting it manually to "2.17.6" fixed all the issues.

**Comment by EmaSuriano** [comment]
Hey, @Aleksefo I saw that this issue is related to a change the team of Contentful did a few versions ago.

I recommend you to use `npm` instead of `yarn`, so then you will get the same I specified versions I used in the `package-lock.json`. 

I created a new instance of the starter and it runs `start` and `build` without any issue.

I will link this issue with this issue and keep an eye when it's fixed. https://github.com/gatsbyjs/gatsby/issues/15344
